earthmover/operations/row.py

```diff
@@ -124,3 +124,48 @@ def execute(self, data: 'DataFrame', **kwargs):
                 raise
 
             return data.sort_values(by=self.columns_list, ascending=(not self.descending))
+
+
+class FlattenOperation(Operation):
+    """
+
+    """
+    allowed_configs: Tuple[str] = (
+        'operation', 'repartition', 
+        'flatten_column', 'left_wrapper', 'right_wrapper', 'separator', 'value_column', 'trim_whitespace'
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Only 'column' or 'columns' can be populated
+        self._flatten_column  = self.error_handler.assert_get_key(self.config, 'flatten_column', dtype=str, required=True)
```

Comment:
I'd recommend making all these instance attributes public instead of private to align with the other Nodes.